L1Trigger/L1THGCal/test/testHGCalL1TBestChoiceTriggerCell_cfg.py

```diff
@@ -25,7 +25,7 @@
 
```

Comment:
Don't modify the best choice tester. Your updates are not related to the front end best choice class.
Revert your changes here.